src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -790,6 +787,10 @@ def test_create_many_groups(self, project_name):
         self._make_crash('u4', state='unreproducible3')
     ]
 
+    # TODO(metzman): Add a seperate test for strategies.
```

Comment:
Instead of moving this, could you add a separate test to ensure that we're able to get the strategy information from the log correctly?